libnavigation-base/api/current.txt

```diff
@@ -493,6 +493,19 @@ package com.mapbox.navigation.base.trip.model.alert {
     method public com.mapbox.navigation.base.trip.model.alert.CountryBorderCrossingAlert.Builder to(com.mapbox.navigation.base.trip.model.alert.CountryBorderCrossingAdminInfo? to);
   }
 
+  public final class CountryBorderCrossingInfo {
```

Comment:
Should we also add this type to `CountryBorderCrossingAlert` instead of separate `to` and `from` fields?

Reply:
done